lib/clouds/shared_functions.py

```diff
@@ -869,6 +872,9 @@ def is_cloud_image_uuid(self, imageid) :
         if self.get_description() == "Parallel Container Manager Cloud" :
```

Comment:
Weren't we going to get these hard-coded cloud sections out of here and into configuration files? Can we do that before merging this?